A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_SelectWptPage.js

```diff
@@ -55,7 +55,7 @@ class A320_Neo_CDU_SelectWptPage {
                     callback(w);
                 };
                 mcdu.onLeftInput[5] = () => {
-                    CDUFlightPlanPage.ShowPage(mcdu, 0);
+                    mcdu.returnCallback(mcdu);
```

Comment:
This is the simplest way I can think of that doesn't involved passing a returnPage callback through a chain of functions.